vm/src/builtins/object.rs

```diff
@@ -329,7 +329,14 @@ impl PyBaseObject {
 
     #[pymethod(magic)]
     fn sizeof(zelf: PyObjectRef) -> usize {
-        zelf.class().slots.basicsize
+        let mut res: usize = 0;
+        let isize: usize = zelf.class().slots.itemsize;
```

Comment:
```suggestion
        let itemsize: usize = zelf.class().slots.itemsize;
```
to avoid reusing primitive type name `isize`